CMP1903_A1_2324/Testing.cs

```diff
@@ -8,13 +9,30 @@ namespace CMP1903_A1_2324
 {
     internal class Testing
     {
-        /*
-         * This class should test the Game and the Die class.
-         * Create a Game object, call the methods and compare their output to expected output.
-         * Create a Die object and call its method.
-         * Use debug.assert() to make the comparisons and tests.
-         */
+        private Game testGame = new Game();
+        private Die testDie = new Die();
+        // Creates a Die and Game object for testing purposes.
 
-        //Method
-    }
+        public void Debugging()
+        // Method for error testing and debugging the Game and Die classes.
+        {
+            Console.WriteLine("Running test..."); 
+            testGame.RollDice(); 
+
+            foreach (int i in Game.values)
+            {
+                Debug.Assert(i >= 1, "Die has rolled below a 1.");
+                Debug.Assert(i <= 6, "Die has rolled above a 6.");
+                // Checks each of the dice rolled and whether or not they're above 6 or below 1.
+            }
+
```

Comment:
Check that you have < and > the right way around for the error message you're showing